Client/Frontend/Browser/Onboarding/Welcome/PlaylistOnboardingView.swift

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveUI
+
+struct PlaylistOnboardingView: View {
+    var onButtonPressed: (() -> Void)?
+    
+    var body: some View {
+        HStack(alignment: .top, spacing: 14.0) {
+            Image("welcome-view-playlist-brave-logo")
+            
+            VStack(alignment: .leading, spacing: 16.0) {
+                VStack(alignment: .leading, spacing: 4.0) {
+                    Text("Ad block is just the beginning")
+                        .font(.title3.weight(.medium))
+                        .foregroundColor(Color(.bravePrimary))
+                    
+                    Text("With Brave Playlist, you can play videos in the background, picture-in-picture, or even offline. And, of course, ad-free.")
+                        .multilineTextAlignment(.leading)
+                        .font(.body)
+                        .foregroundColor(Color(.bravePrimary))
+                }
+                .accessibilityElement(children: .combine)
+                
+                Button(action: { onButtonPressed?() }) {
+                    HStack {
+                        Image("welcome-view-playlist-play-icon")
+                        Text("Watch the video")
+                            .font(.title3.weight(.medium))
+                            .foregroundColor(Color(.braveBlurple))
+                    }
+                }
+            }
+        }
+        .frame(maxWidth: 450)
+        .padding(.all)
```

Comment:
nit: `.all` can be removed.

Reply:
Fixed at https://github.com/brave/brave-ios/pull/4508/commits/a2e917b40f152ee684381668473bb9b7338567c1